app/src/lib/git/rebase.ts

```diff
@@ -546,9 +541,10 @@ export async function rebaseInteractive(
   repository: Repository,
   pathOfGeneratedTodo: string,
   lastRetainedCommitRef: string | null,
-  action: string = 'interactive rebase',
+  action: string = 'Interactive rebase',
   gitEditor: string = ':',
-  progressCallback?: (progress: IRebaseProgress) => void
+  progressCallback?: (progress: IMultiCommitOperationProgress) => void,
+  commits?: ReadonlyArray<Commit>
```

Comment:
Decided to pass commits in because in the squash method and likely for reorder we have to obtain this list of commits to generate  the todo list. -> no need to do the git command again if we can just pass in the commits.